### ISSUE TITLE
Meta-issue for sdo-callisto release (likely as version 3.2)

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#1292]
Targetting for publication: February 2017. 

See http://webschemas.org/docs/releases.html#sdo-callisto for proposed public "release notes". 

Implemented
- https://github.com/schemaorg/schemaorg/issues/1294 - improvements around 'material' (implemented).
- https://github.com/schemaorg/schemaorg/pull/1302 - fixed a problem with hierarchical listing not sticking to correct extension layers.
- /sameAs "Freebase" -> "Wikidata"; #1162
-  accessMode, accessModeSufficient, and accessibilitySummary properties for e.g. epubs -  https://github.com/schemaorg/schemaorg/issues/1100

Site infrastructure
- #1285
-

Final checklist / @danbri tasks
- [x] double-check all changes are listed here and in releases.html as appropriate.
- [x] update language in doc/*.html to clarify that when we say "sponsor" we mean "founder" organizations.
- [ ] update aboutus.html for recent changes.
- [x] Check that Courses, Channel, Menu, ClaimReview are properly documented.
- [ ] Fix/change the ClaimReview example.
- [x] Add a structure to releases.html that clarified moves from pending to core or other hosted extensions, and that introduces 'attic' for retired ideas.
- [x] Check for final "low hanging fruit"
- [x] Fibo/Finance and guidance for hosted extension maintainance.
- [ ] Merge PR #1539 - examples for the new Consortium Type added to pending in this release.
- [x] Merge PR #1489 examples for the newly added identifier property.
- [x] Merge PR #1475 add Event to domain of audience 
- [x] Merge PR #1443 - fix a link that 404’s in the docs/schemas.html page
- [x] Merge PR #1416 - acknowledge the support of the Legal Entity Identifier Foundation
- [x] Circulate to CG+SG for final review.

**Comment by gmackenz** [comment]
If possible, I'd like to add #1288 - Proposal of a Menu/menuItems schema as a possible inclusion to sdo-callisto.

**Comment by twamarc** [comment]
@danbri is it possible to add also the 1) https://github.com/schemaorg/schemaorg/issues/492#issuecomment-233254224
and the 2)https://github.com/schemaorg/schemaorg/issues/1114
in this coming release? The é topics have been largely discussed and I think we have now a consensus on them:
* (for 1) We will only extend the definition to cover both person and place as suggested @westurner in https://github.com/schemaorg/schemaorg/issues/492#issuecomment-233419164 but we will foresee some additional  properties under it as suggested by @LezzaRodrigues  in https://github.com/schemaorg/schemaorg/issues/492#issuecomment-214068477. 

*(for 2) https://github.com/schemaorg/schemaorg/issues/1114#issuecomment-212845014).

~Marc (for ScheMed W3C WG) and health-lifesci.schema.org extension.

**Comment by danbri** [comment]
Also relevant, please note the summary @RichardWallis has been updating for tools/python/infrastructure-related (rather than vocab) issues, see #3.

**Comment by danbri** [comment]
I've been updating docs/releases.html towards having a release candidate. 

http://webschemas.org/docs/releases.html

As you can see, we have been filling the "pending" pipeline. I've started restructuring the page to have a way to show progression of successful ideas into e.g. Core section, or abandoned ideas into the "Attic". It's not ideal yet but I believe all the basic content should be there now.

**Comment by danbri** [comment]
TODO: we are missing some non-vocabulary improvements, specifically Travis-CI unit testing + the structured data checker being run over our examples.

**Comment by danbri** [comment]
TODO: http://localhost:8080/accessMode etc. also migrate to Core.

**Comment by rtroncy** [comment]
... and you want to correct in http://webschemas.org/docs/releases.html that sdo-callisto is due by
**(2017-03-XX)** (and not 2016)

**Comment by gkellogg** [comment]
> TODO: we are missing some non-vocabulary improvements, specifically Travis-CI unit testing + the structured data checker being run over our examples.

To be clear, we do have the unit testing, and are running code based on the linter over all examples on Travis-CI, we're just ignoring the results. The work to do is in applying the test results to the examples themselves. I've corrected most syntactic errors, what remains is semantic errors (use of obsolete terms, for example). The plan was to work away at these, and enable the Travis-CI tests to fail when an error is found. This requires some judgement call, which I was not comfortable making unilaterally.

**Comment by danbri** [comment]
@gkellogg - yes, exactly. It is one thing to be checked, it is another to get the all clear. For our next release I'd love to focus on that kind of cleanup.

**Comment by danbri** [comment]
Release candidate circulated: https://lists.w3.org/Archives/Public/public-schemaorg/2017Mar/0013.html PTAL

**Comment by danbri** [comment]
Also needed: 2-3 sentence release summary with main highlights.

**Comment by eliaskaerle** [comment]
Pls. consider integrating #1259 into the release. There was no further complaint or suggestion since Oct. '16.

**Comment by sumutcan** [comment]
Please consider the change about the occupancy property (#1524), I think this is one of the low hanging fruits.

**Comment by NicoSki** [comment]
As see that the SkiResort extention is not in the newest release. Being a wintersports journalist I know that travellers are very keen on information about the number of lifts, the number of skislopes and the length of every lift or slope. Would it be possible to add that kind of information as well to schema.org? 
with kind regards
Nico

**Comment by chaals** [comment]
I think we should try to get this release out. I have made minor comments on the skiResort stuff, which I think would be good to get out, and I have some concerns about `speakable` which I should either try to fix or decide it's OK really quickly…

**Comment by danbri** [comment]
Can you articulate the issue with speakable? It is going only into pending at this stage, i.e. "kick the tires" mode

**Comment by AymenLoukil** [comment]
Release details page is missing for 3.2
https://github.com/schemaorg/schemaorg/issues/1544

**Comment by AymenLoukil** [comment]
@danbri Sitemap.xml added by @RichardWallis too, to mention
https://github.com/schemaorg/schemaorg/issues/1483

**Comment by Dataliberate** [comment]
Sitemap.xml is available: http://webschemas.org/docs/sitemap.xml
It won't fully reflect the recent changes, as the process to create it is only run as part of the final release process.

On the _webschemas_ site however it is not referenced in the robots.txt file - as we don't want to facilitate crawling of this work-in-progress site.

When released on the schema.org site it will be picked up via this line in robots.txt:
```Sitemap: http://schema.org/docs/sitemap.xml```

**Comment by chaals** [comment]
The `speakable` sounds wrong to me, since it's sort of a summary, but for kicking its tires I can live with it - and I suspect in the end I will just think that name was poorly chosen as too tied to one of several specific use cases.

So let's publish. In particular, people are waiting on the accessibility improvements.

**Comment by danbri** [comment]
Ok, let's do it. I'll fold in the most obvious things from above and open a new issue for a next release. It was interesting to leave it a few months collecting ideas in pending.webschemas.org but it'd be healthier to get back on a tighter release cycle again.

**Comment by danbri** [comment]
Release in progress, aimed at Thursday, DNS changes for new subdomains should be propagating now. Current release candidate is http://3-2-1.webschemas-g.appspot.com/docs/releases.html - anything from above that we didn't get to is first on the list for attention in next release.

**Comment by danbri** [comment]
http://blog.schema.org/2017/03/schemaorg-32-release-courses-fact.html

...and that's a wrap! Thanks all :)
